lib/meadow/data/schemas/work_descriptive_metadata.ex

```diff
@@ -118,7 +118,15 @@ defmodule Meadow.Data.Schemas.WorkDescriptiveMetadata do
     def encode_field([field | []]), do: [encode_field(field)]
     def encode_field([field | fields]), do: [encode_field(field) | encode_field(fields)]
 
-    def encode_field(%ControlledMetadataEntry{} = field), do: Map.from_struct(field)
+    def encode_field(%ControlledMetadataEntry{role: nil} = field) do
+      Map.from_struct(field)
+      |> Map.put(:display_facet, field.term.label)
```

Comment:
Should this be camelCased as `displayFacet` to match the other fields?

Reply:
argh! yes. Thanks!

Reply:
No problem! Not sure if that'll affect the `*facet` change in `meadow.json` or not, but you may want to verify.

Reply:
arg. yes it will. thanks.